commission/reports/report_settlement_templates.xml

```diff
@@ -51,21 +51,31 @@
                                 </td>
                             </tr>
                         </tbody>
-                        <tfoot>
-                            <tr>
-                                <td />
-                                <td class="text-end">
-                                    <strong>Total</strong>
-                                </td>
-                                <td class="text-end">
-                                    <span
-                                        t-field="o.total"
-                                        t-options="{'widget': 'monetary', 'display_currency': o.currency_id}"
-                                    />
-                                </td>
-                            </tr>
-                        </tfoot>
                     </table>
+                    <div class="clearfix mb-4">
+                        <div id="total" class="row">
+                            <div
+                                t-attf-class="#{'col-6' if report_type != 'html' else 'col-sm-7 col-md-6'} ms-auto"
+                            >
+                                <table
+                                    class="table table-sm table-borderless"
+                                    style="page-break-inside: avoid;"
+                                >
+                                    <tr class="border-black o_total">
+                                        <td>
+                                            <strong>Total</strong>
+                                        </td>
+                                        <td class="text-end">
+                                            <span
+                                                t-field="o.total"
```

Comment:
Isn't `total` already a monetary field for avoiding to put the `t-options`? And if not, better to convert it to monetary.